packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -17,7 +17,7 @@ import { VideoTile, VideoTileStylesProps } from './VideoTile';
 export const LocalVideoTile = React.memo(
   (props: {
     userId: string;
-    onCreateLocalStreamView?: (options?: VideoStreamOptions) => Promise<void>;
+    onCreateLocalStreamView?: (options?: VideoStreamOptions) => Promise<void | CreateVideoStreamViewResult>;
```

Comment:
Just curious, will this still work when we usePropsFor, wondering if auto complete is still working fine for this

Reply:
It should JustWork, here's what the prop looks like:
![image](https://user-images.githubusercontent.com/2684369/170103900-1c680bca-0498-4daa-9131-ff4121cd4e7c.png)